services/src/main/java/org/exoplatform/webconferencing/externalvisio/ExternalVisioProvider.java

```diff
@@ -67,4 +75,13 @@ public String[] getSupportedTypes() {
   public UserInfo.IMInfo getIMInfo(String imId) throws CallProviderException {
     return null;
   }
+
+  @Override
+  public List<ActiveProviderInfo> getActiveProvidersForSpace(String spaceId) {
+    List<ExternalVisioConnector> externalVisioConnectors = externalVisioConnectorService.getActiveExternalVisioConnectorsForSpace();
+
```

Comment:
One thing is missing here : 
for each externalVisioConnector, you need to get the associated setting for the current space